scripts/check_maps.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
```

Comment:
This is a checker script that goes through "triplea_maps.xml" and verifies that each URL listed as available for a  map download, is actually available.